models/measure.js

```diff
@@ -121,7 +121,7 @@ module.exports = (event, state) => {
         loading: { ...state.loading, page: isMeasureDetailPage(state.location.route) ? false : state.loading.page, measure: false },
         location: {
           ...state.location,
-          title: isMeasureDetailPage(state.location.route) ? `${event.measure.legislature_name}: ${event.measure.title}` : state.location.title,
+          title: isMeasureDetailPage(state.location.route) ? `${hideTargetReps ? '' : `${event.measure.legislature_name}: `}${event.measure.title}` : state.location.title,
```

Comment:
For this check maybe the function should be called something like `hideLegislatureName`? `hideTargetReps` doesn't make sense on this page because there is no `targetReps` here.